src/js/modules/popupRegistration.js

```diff
@@ -47,18 +40,15 @@ function submitHandler() {
 	if (!registerModalOverlay.classList.contains("open")) {
```

Comment:
looks like you can move `registerModalContent.classList.remove("open");` into the condition, don't you?
if there is no opened registerModalOverlay then we don't have opened registerModalContent, right?

Reply:
In this case it won't work properly. I think you're mistaken.

Reply:
yes, I am wrong, I didn't think about the case when the event was caused in the modal itself